src/main/java/com/binance/api/client/impl/BinanceApiService.java

```diff
@@ -61,9 +61,15 @@
 
   @GET("/api/v1/ticker/24hr")
   Call<TickerStatistics> get24HrPriceStatistics(@Query("symbol") String symbol);
+  
+  @GET("/api/v1/ticker/24hr")
+  Call<List<TickerStatistics>> getAll24HrPriceStatistics();
 
   @GET("/api/v1/ticker/allPrices")
   Call<List<TickerPrice>> getLatestPrices();
+  
+  @GET("/api/v3/ticker/price")
+  Call<TickerPrice> getSymbolPrice(@Query("symbol") String symbol);
```

Comment:
For consistency, can you please call this `getLatestPrice`?

Reply:
done